onchain/rollups/test/foundry/consensus/quorum/QuorumFactory.t.sol

```diff
@@ -0,0 +1,161 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+pragma solidity ^0.8.8;
+
+import {Vm} from "forge-std/Vm.sol";
+
+import {QuorumFactory} from "contracts/consensus/quorum/QuorumFactory.sol";
+import {Quorum} from "contracts/consensus/quorum/Quorum.sol";
+import {History} from "contracts/history/History.sol";
+import {IHistory} from "contracts/history/IHistory.sol";
+
+import {TestBase} from "../../util/TestBase.sol";
+
+import "forge-std/console.sol";
+
+contract QuorumFactoryTest is TestBase {
+    QuorumFactory factory;
+
+    event QuorumCreated(address[] quorumValidators, Quorum quorum);
+
+    struct QuorumCreatedEvent {
+        address[] quorumValidators;
+        Quorum quorum;
+    }
+
+    function setUp() public {
+        factory = new QuorumFactory();
+    }
+
+    function testNewQuorum(uint256 _numValidators) public {
+        _numValidators = bound(_numValidators, 2, 50);
+        vm.assume(_numValidators >= 2 && _numValidators <= 50);
```

Comment:
This check is not necessary, as this is guaranteed by `bound`.
This also applies to `testNewQuorumDeterministic`.